modules/core/src/main/java/org/apache/ignite/internal/processors/tracing/SpanManager.java

```diff
@@ -29,7 +29,7 @@
      *
      * @param spanType Type of span to create.
      */
-    public default Span create(@NotNull SpanType spanType) {
```

Comment:
By the way, do we need two separate abstractions: Tracing and SpanManager?
As long as Tracing SPI implements neither of them, looks like we can keep only one.

Reply:
I think yes. Tracing is general one that encapsulates both SpanManager and TracingConfiguratoin. We might prefer composition over inheritance here, but I'm not sure that it's necessary. Probably when will will add some mote stuff to Tracing we might move SpanManager inside Tracing it instead of inheriting it.